src/info_parser.py

```diff
@@ -0,0 +1,150 @@
+                "INFOPLIST_FILE": "custom/{}".format(self.info_plist_path()),
+                "INFOPLIST_KEY_CFBundleDisplayName": self._app_name(),
+                "INFOPLIST_KEY_UILaunchStoryboardName": "Launch.storyboard"
+            }
+            },
+            "configFiles": {
+                "Debug": self._xcconfig_full_path(),
+                "Release": self._xcconfig_full_path()
+            },
+            "sources": [
+                {"path": "custom/{}".format(self.brand_name)},
+                {"path": "custom/Launch.storyboard",
+                 "destinationFilters": ["iOS"]
+                 },
+                {"path": "Support",
+                 "excludes": [
+                     "*.xcassets",
+                     "Info.plist"
+                 ] + self._excluded_languages()
+                 },
+            ]
+        }
+        return {self.brand_name: dict}
+    
+    def zimurl(self):
+        return self.data[JSON_KEY_ZIM_URL]
+    
+    def zim_file_path(self):
+        url = self.zimurl()
+        return "{}/{}".format(self.brand_name, os.path.basename(url))
+    
+    def download_auth(self):
+        auth_key = self.data[JSON_KEY_AUTH]
+        return os.getenv(auth_key)
+
+    @staticmethod
+    def plist_commands():
+        for value in (JSON_TO_XCCONFIG_MAPPING.values()):
+            if value != "APP_STORE_ID":
+                yield InfoParser._add_to_plist_cmd(value)
+        yield InfoParser._add_to_plist_cmd(XCCONF_KEY_ZIM_FILE)
+
+    # private
+    @staticmethod
+    def _add_to_plist_cmd(value):
+        return "/usr/libexec/PlistBuddy -c \"Add :{} string \$({})\"".format(value, value)
+
+    def _app_version(self):
+        return self._app_version_from(self.zim_file_name)
+
+    def _app_name(self):
+        return self.data[JSON_KEY_APP_NAME]
+
+    def _enforced_language(self):
+        if JSON_KEY_ENFORCED_LANGUAGE in self.data:
+            return self.data[JSON_KEY_ENFORCED_LANGUAGE]
+        else:
+            return None
+
+    def _brandname_from(self, filepath):
+        return os.path.basename(os.path.dirname(filepath)).lower()
+
+    def _filename_from(self, url):
+        return os.path.splitext(os.path.basename(urlparse(url).path))[0]
+
+    def _xcconfig_full_path(self):
+        return "custom/{}".format(self.xcconfig_path())
+
+    def _app_version_from(self, file_name):
+        m = re.search('\d{4}-\d{1,2}', file_name)
+        yearMonth = m.group(0)
+        (year, month) = map(lambda x: int(x), yearMonth.split("-"))
+        assert (year > 2000)
+        assert (month > 0)
+        assert (month <= 12)
+        return ".".join([str(year), str(month)])
+
+    def _excluded_languages(self):
+        enforced = self._enforced_language()
+        if enforced == None:
+            return ["**/qqq.lproj"]
+        else:
+            excludes = []
+            # exclude every lproj folder except the enforced one
```

Comment:
This part makes sure we are fixing: https://github.com/kiwix/apple/issues/603
Meaning, if we are enforcing a language, we exclude all other language files from the project.